src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
+
+    public static function getName(): string
+    {
+        return 'schedule-executions';
+    }
+
+    public static function getSupportedResource(): string
+    {
+        return 'execution';
+    }
+
+    protected function enqueueResources(Group $pools, Database $dbForConsole): void
+    {
+        $queue = $pools->get('queue')->pop();
+        $connection = $queue->getResource();
+        $queueForFunctions = new Func($connection);
+
+        foreach ($this->schedules as $schedule) {
+            if (!$schedule['active']) {
+                unset($this->schedules[$schedule['resourceId']]);
+                continue;
+            }
+
+            $now = new \DateTime();
+            $scheduledAt = new \DateTime($schedule['schedule']);
+
+            if ($scheduledAt > $now) {
+                continue;
+            }
+
+            $queueForFunctions
+                ->setType('schedule')
+                ->setFunctionId($schedule['resource']['functionId'])
```

Comment:
Let's add a comment here, that we only set ID because we don't have dbForProject at this point. Worker will later get the whole function.

Reply:
I would like to use dbForPRoject here, but this shouldn' be the reason to introduce it. When we have a comment, in future, when we add dbForProject for other reasons, we will improve this too.

Reply:
done